lib/web-portal.ts

```diff
@@ -84,7 +91,7 @@ export class WebPortalStack extends cdk.NestedStack {
         })
 
         // redirect /static/tunasync.json to /jobs
-        props.externalALBListener.addAction(`${usage}RedirectHTTPAction`, {
+        props.externalALBListener.addAction(`${usage}RedirectJobsHTTPAction`, {
```

Comment:
Renaming the name of CFN ALBListener resource has to change the priority accordingly, otherwise 
the updating of stack will fail on the conflict priority value.